pkg/operator/ceph/file/mds/config.go

```diff
@@ -71,8 +71,11 @@ func (c *Cluster) setDefaultFlagsMonConfigStore(mdsID string) error {
 	configOptions := make(map[string]string)
 
 	// Set mds cache memory limit to the best appropriate value
-	if !c.fs.Spec.MetadataServer.Resources.Limits.Memory().IsZero() {
-		mdsCacheMemoryLimit := float64(c.fs.Spec.MetadataServer.Resources.Limits.Memory().Value()) * mdsCacheMemoryLimitFactor
+	if !c.fs.Spec.MetadataServer.Resources.Requests.Memory().IsZero() {
```

Comment:
Based on discussion in #8143, how about the following approach?
1. If the limits are set, we have the previous behavior to set `mds_cache_memory_limit` by multiplying by `mdsCacheMemoryFactor`
2. If the limits are not set and the requests are set:
   - Set the `mds_cache_memory_limit` **without** multiplying by `mdsCacheMemoryFactor`

Reply:
going with this approach discussed [here](https://github.com/rook/rook/issues/8143#issuecomment-869956354)

```
mdsCacheMemoryLimitFactor = 0.5
mdsCacheMemoryRequestFactor = 0.8```